.travis.yml

```diff
@@ -1,11 +1,23 @@
+arch: arm64-graviton2
```

Comment:
Travis CI docs/in-app messaging encourage folks to try `arch: arm64-graviton2`, saying builds might be faster. Might be fake news; the Graviton2 EC2 instances are a better value than amd64 ones so it makes sense they'd push folks to use it.

Reply:
Ooh, interesting.